### PR TITLE
[T4A6][T17-B1]Chia Zhe Min

### PR DESCRIPTION
**Pull request by zhems**

### CODE REVIEW

**Comment by zhems** [comment]
Ready for review

File: src/seedu/addressbook/common/UtilsTest.java [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+import java.util.ArrayList;
+
+import static org.junit.Assert.*;
+import static seedu.addressbook.common.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.addressbook.common.Messages.MESSAGE_INVALID_PERSON_DISPLAYED_INDEX;
+
+public class UtilsTest {
+
+    @Test
+    public void isAnyNull_returnsFalse() throws IllegalValueException {
```
Comment (JoelT-92) [inline src/seedu/addressbook/common/UtilsTest.java:26]: Remember that the naming convention is TestName_Scenario_Result, or just TestName if you want to widen the scope of the test.

File: src/seedu/addressbook/common/UtilsTest.java [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+        Email email = new Email("chiazhemin@gmail.com",true);
+        Email email2 = new Email("jameschia@gmail.com",true);
+        assertFalse("2 emails, no null",Utils.isAnyNull(email,email2));
+    }
+    
+    @Test
+    public void isAnyNull_returnsTrue() throws IllegalValueException {
+        Email email = new Email("chiazhemin@gmail.com",true);
```
Comment (JoelT-92) [inline src/seedu/addressbook/common/UtilsTest.java:34]: Noticed that most of the code is reused. You may wish to refactor the code by extracting another method out!

File: src/seedu/addressbook/common/UtilsTest.java [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+    @Test
+    public void isAnyNull_returnsTrue2() {
+        assertFalse("empty arguments",Utils.isAnyNull());
+    }
+    
+    @Test
+    public void elementsAreUnique_returnTrue() throws IllegalValueException {
+        
+        List<Email> emailList = new ArrayList<>();
+        emailList.add(new Email("chiazhemin@gmail.com",true));
```
Comment (JoelT-92) [inline src/seedu/addressbook/common/UtilsTest.java:49]: Magic strings could have been refactored as well.

**Comment by JoelT-92** [comment]
@zhems Some comments added. Please close the PR after reading comments.

**Pull request closed without merging**